Add guidance for managing high-value vendor contracts (#18239)
PostHog/posthog.com: contents/handbook/people/spending-money.md

Joining an offsite? _Only use the offsite budget_, not your User Limit - it helps the People & Ops track travel spend accurately against budgets. Let Kendal know in [#team-people-and-ops](https://posthog.slack.com/archives/C017WDX3BFZ).

## High-value & ongoing vendor contracts

Some of what we spend money on isn't a personal subscription or a one-off purchase - it's a platform a team runs to do its work, on a contract that bills continuously and at scale. Think infrastructure like Kafka, or certain security tooling. Spend here can run into the thousands or hundreds of thousands, so it deserves more care than "I subscribed to XYZ" - the guiding principles above still apply, they just matter more.

This section is about _living with_ a contract we've already got. Deciding whether to adopt a new tool in the first place is covered separately in [Adding tools](/handbook/company/adding-tools).

> **Who owns it?** The team that operates the vendor owns its quota and spend - in practice, the person with the most context about the contract, usually whoever proposed or operates the tool. That includes keeping an eye on usage against the contract's limits (the usage terms and limits you agreed when you signed it) so nobody is surprised by the bill. The easiest way to stay on top of this is to fold it into the vendor's own periodic review (e.g. a quarterly business review), so quota and spend get looked at on a regular cadence rather than only when something goes wrong. Loop in your exec early - they should have enough context to support and guide you.

Reviewer: The owner should be an individual and they should be the person that signs the contract.

> **Who owns it?** The team that operates the vendor owns its quota and spend - in practice, the person with the most context about the contract, usually whoever proposed or operates the tool. That includes keeping an eye on usage against the contract's limits (the usage terms and limits you agreed when you signed it) so nobody is surprised by the bill. The easiest way to stay on top of this is to fold it into the vendor's own periodic review (e.g. a quarterly business review), so quota and spend get looked at on a regular cadence rather than only when something goes wrong. Loop in your exec early - they should have enough context to support and guide you.

Author: ```suggestion
**Who owns it?** The team that operates the vendor owns its quota and spend - in practice, the owner should be an individual from the owning team and they should be the person that signs the contract which usually is the person with the most context about the contract and/or proposed or operates the tool. All that includes keeping an eye on usage against the contract's limits (the usage terms and limits you agreed when you signed it) so nobody is surprised by the bill. The easiest way to stay on top of this is to fold it into the vendor's own periodic review (e.g. a quarterly business review), so quota and spend get looked at on a regular cadence rather than only when something goes wrong. Loop in your exec early - they should have enough context to support and guide you.
```

Author: @fraserhopper wdyt?

Reviewer: yeah lgtm

- The person with the most context on the contract drives this. Keep your team lead and the responsible exec aware - they own the area and help decide whether the extra spend is worth it or whether we right-size.
- Ping `@finance-folks` in Slack as soon as the overage looks material, and take it from there. That's usually enough - it keeps Finance in the loop so they can forecast it accurately.
- Pull in [#legal](https://posthog.slack.com/archives/C08MYQX74KH) if this is a _new_ contract, or if the legal details have changed and need a second look. Straight re-commits or renewals usually don't need legal, but a new contract always does (see [Adding tools](/handbook/company/adding-tools)).
- For company-level tool billing, [Janani](https://posthog.com/community/profiles/34497) in [#team-people-and-ops](https://posthog.slack.com/archives/C017WDX3BFZ) is the default admin, so billing stays owned and categorized correctly. The billing email for any vendor contract is finance@posthog.com.

> There's no fixed dollar amount that trips a formal approval - we stay trust-based and lead with context, not permission. But spend at this scale isn't comparable to a personal subscription, so be proactive and transparent about it _early_ rather than once the overage has already landed.

Reviewer 2: ```suggestion
There's no fixed dollar amount that trips a formal approval - we stay trust-based and lead with context, not permission. Use your judgement for what counts as material. But spend at this scale isn't comparable to a personal subscription, so be proactive and transparent about it _early_ rather than once the overage has already landed.
```

  - Add [Janani](https://posthog.com/community/profiles/34497) as the Billing Admin to manage payments.
- If I'm asked for a billing email – for bill payments or when signing a vendor contract – what do I use?
  - Use finance@posthog.com
- A vendor contract we rely on is running into overages / about to cost more – what do I do?

>   - Flag it early - see [High-value & ongoing vendor contracts](#high-value--ongoing-vendor-contracts). Whoever has the most context on the contract should drive, keep your team lead and the responsible exec aware, and ping `@finance-folks` in Slack as soon as the overage looks material so Finance can forecast it.

Reviewer 2: ```suggestion
  - Flag it early - see [High-value & ongoing vendor contracts](#high-value--ongoing-vendor-contracts). Whoever has the most context on the contract should drive, keep your team lead and the responsible exec aware, and ping `@finance-folks` in Slack as soon as the overage looks material so Finance can forecast it. Use your judgement to assess what counts as material.
```
